romanian/padurea_violet/padurea_violet.md

```diff
@@ -1,3 +1,11 @@
 Au intrat in padurea violet si dupa 2 ore au ajuns intr-o poiana.
 Aici au intalnit un :
-  - [crocobalaur](croco/croco.md) 
\ No newline at end of file
+  - [crocobalaur](croco/croco.md) , 
+
+care salasluia in fitocenozele dominate de Potamogeton gramineus, ce sunt identificate în
+Lunca  şi Delta Dunării, precum şi în bazinele din cursul Brăilei. Se dezvoltă în ape cu adâncimi variabile (0,2 ‐ 1,5m) oligo‐ sau mezotrofe, sărace în calcar şi pe un substrat
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.